### PR TITLE
Modify webkit-release script so it can be used for WPE

### PR DESCRIPTION
**Pull request by aperezdc**
There's more tweaks to be done, but this is a reasonable first pass which allowed me to already try `install-tarball`, `send-announcement`, and `gen-tarball-meta` for WPE tarballs.

### CODE REVIEW

File: webkit-release [hunk 1]
```diff
@@ -778,6 +835,9 @@ if __name__ == '__main__':
         prog = 'webkit-release',
         usage = '%(prog)s [options ...] command [command-options ...] tarball')
 
+    parser.add_argument('-p', '--port', dest = 'port', default = "gtk", choices = ("gtk", "wpe"),
```
Comment (carlosgcampos) [inline webkit-release:875]: I think it should be easy to autodetect the port form the tarball instead of having to pass --port to the script

**Review by carlosgcampos — CHANGES_REQUESTED** [review]
LGTM, but I would avoid adding the command line option if possible.

**Review by carlosgcampos — COMMENTED** [review]
Why doesn't WPE include a NEWS file?

File: webkit-release (hunk 1 above)
Comment (aperezdc) [reply webkit-release:875]: Sure, I can look into doing that. Should this be detected based on the tarball name, or from its contents? I think the tarball names will always match `^wpewebkit-.*` and `^webkitgtk-.*` so probably we don't need anything more complicated that checks for certain files inside the tarball.

Comment (carlosgcampos) [reply webkit-release:875]: Whatever is easier for you :-)

**Comment by aperezdc** [comment]
@carlosgcampos I am using the `gen-tarball-meta` subcommand with the `wpebackend-*.tar.xz` tarballs, which I think didn't have a `NEWS` file... but eventually we added them. Probably it's better to remove handling the missing `NEWS` files, so I'll do that.

**Comment by aperezdc** [comment]
I have a [working on solution](https://git.sr.ht/~aperezdc/webkit-releng-toolbelt) which supports both WebKit ports, *plus* libwpe + WPEBackend-fdo + Cog. More kinds of packages can be easily supported adding a template for them. 

So far it only supports extracting the `NEWS` from tarballs, generating text output from templates (`.sums`, `.news`, announcement e-mail, blog post), and signing tarballs. While it does not yet support all the `webkit-release` features, I have been using it just fine for the WPE components.

Closing this as it's obsoleted by the new `wkrel` script linked :wink:

**Pull request closed without merging**